src/views/components/landing-top.html

```diff
@@ -337,6 +208,228 @@ <h3>over {{_.formatMiB2TiB(networkStats.megabytes.total)}} TiB</h3>
 					{ slug: 'gh', label: 'GitHub', image: 'github.svg' },
 					{ slug: 'wp', label: 'WordPress', image: 'wordpress.svg' },
 				],
+				slidesData: {
```

Comment:
Why this change? It's probably harder to edit than before.

Reply:
I believe that it is much easier to edit the data or extend the functionality if needed later when the data and the view are separated